spec/locale.html

```diff
@@ -136,7 +136,8 @@ <h1>
           1. Let _optionsValue_ be _options_.[[&lt;_key_&gt;]].
           1. If _optionsValue_ is not *undefined*, then
             1. Assert: Type(_optionsValue_) is String.
-            1. Set _value_ to _optionsValue_.
+            1. Let _optionsUValue_ be the ASCII-lowercase of _optionsValue_.
+            1. Set _value_ to the String value resulting from canonicalizing _optionsUValue_ as a value of key _key_ per <a href="https://unicode.org/reports/tr35/#processing-localeids">Unicode Technical Standard #35 Part 1 Core, Annex C LocaleId Canonicalization Section 5 Canonicalizing Syntax, Processing LocaleIds</a>.
```

Comment:
[ResolveLocale](https://tc39.es/ecma402/#sec-resolvelocale) also explicitly converts `key` to ASCII lower-case before this step. Should we do the same here? (`key` is already in ASCII lower-case, so it's not strictly required to perform any case conversions, neither here nor in `ResolveLocale`.)

Reply:
In here, Inside 14.1.3 ApplyUnicodeExtensionToTag ( tag, options, relevantExtensionKeys )
https://tc39.es/ecma402/#sec-apply-unicode-extension-to-tag
5. For each element key of relevantExtensionKeys, do

and only place call it is in 
14.1.1 Intl.Locale ( tag [ , options ] )
https://tc39.es/ecma402/#sec-Intl.Locale
29. Let r be [ApplyUnicodeExtensionToTag](https://tc39.es/ecma402/#sec-apply-unicode-extension-to-tag)(tag, opt, relevantExtensionKeys).

where relevantExtensionKeys is from 
2. Let relevantExtensionKeys be [%Locale%](https://tc39.es/ecma402/#sec-intl-locale-constructor).[[RelevantExtensionKeys]].

and  [%Locale%](https://tc39.es/ecma402/#sec-intl-locale-constructor).[[RelevantExtensionKeys]] is defined in 
https://tc39.es/ecma402/#sec-intl.locale-internal-slots
"The value of the [[RelevantExtensionKeys]] internal slot is « "ca", "co", "hc", "kf", "kn", "nu" ». If [%Collator%](https://tc39.es/ecma402/#sec-the-intl-collator-constructor).[[RelevantExtensionKeys]] does not contain "kf", then remove "kf" from [%Locale%](https://tc39.es/ecma402/#sec-intl-locale-constructor).[[RelevantExtensionKeys]]. If [%Collator%](https://tc39.es/ecma402/#sec-the-intl-collator-constructor).[[RelevantExtensionKeys]] does not contain "kn", then remove "kn" from [%Locale%](https://tc39.es/ecma402/#sec-intl-locale-constructor).[[RelevantExtensionKeys]]."

I agree with you that the "converts key to ASCII lower-case" in [ResolveLocale](https://tc39.es/ecma402/#sec-resolvelocale) is also unnecessary. But that is another editoral issue we can change separately